File: C/src/1008.c
#include <stdio.h>

int main(){
    int hour,number;
    float pay;
    scanf("%i\n%i\n%f",&number,&hour,&pay);
    printf("NUMBER = %i\n",number);
    printf("SALARY = U$ %.2f\n",hour*pay);
    return 0;
}

File: go/1008.go
package main

import "fmt"

func main() {
	var number, hour int
	var price float64
	fmt.Scan(&number)
	fmt.Scan(&hour)
	fmt.Scan(&price)
	fmt.Printf("NUMBER = %v\n", number)
	var salary float64 = float64(hour) * price
	fmt.Printf("SALARY = U$ %.2f\n", salary)
}
